### PR TITLE
Update Slurm logo to remove "workload manager" text.

### PR DESCRIPTION
**Pull request by wickberg**

### CODE REVIEW

File: hosted_logos/slinky.svg
Comment (naskio) [inline hosted_logos/slinky.svg:1]: This file shouldn't be included in this PR

File: landscape.yml
Comment (naskio) [inline landscape.yml:1]: Same for these changes

**Review by naskio — CHANGES_REQUESTED** [review]
Hi @wickberg,
Thank you again for your contributions! To proceed with merging, we need to ensure that this PR includes only the changes related to the Slurm logo.

File: hosted_logos/slinky.svg
Comment (wickberg) [reply hosted_logos/slinky.svg:1]: Sorry about that, moved main on my fork without realizing it'd trip this up.

**Comment by wickberg** [comment]
Should be fixed now. I was trying to avoid issues between this and #2 but got the structure a bit wrong. That'll teach me to write these in the middle of KubeCon.

**Review by naskio — APPROVED** [review]
Thanks, LGTM !
